fatf/transparency/models/feature_influence.py

```diff
@@ -623,3 +644,406 @@ def partial_dependence(dataset: np.ndarray,
+                        ' will be used as scoring metric.', UserWarning)
+                else:
+                    pass
+            else:
+                if scoring_metric is None:
+                    if as_regressor is True:
+                        warnings.warn(
+                            'As scoring_metric was not'
+                            ' specified,'
+                            ' max_error will be used as'
+                            ' scoring metric.', UserWarning)
+                    elif as_regressor is False:
+                        warnings.warn(
+                            'As scoring_metric'
+                            ' was not specified,'
+                            ' accuracy will be used as'
+                            ' scoring metric.', UserWarning)
+
+    is_input_ok = True
+    return is_input_ok
+
+
+def get_scores(dataset: np.ndarray,
+               model: object,
+               target: np.ndarray,
+               as_regressor: bool,
+               scoring_metric: Optional[str] = None):
+    """
+    Returns the difference between the trained model's predictions
+    on the ``dataset`` and the ground truth ``target`` according to a
+    particular ``scoring_metric``. For the exceptions description please
+    see the documentation of the
+    :func`fatf.transparency.model.feature_influence.
+    permutation_feature_importance` function.
+    Parameters
+    ----------
+    dataset : numpy.ndarray
+        A dataset based on which model predictions and associated
+        predictive error will be alculated
+    model
+        A fitted model whose predictions will be used to calculate
+        associated predictive error based on 'scoring_metric'. (Please
+        see :class:`fatf.utils.models.models.Model` class documentation for the
+        expected model object specification.)
+    target: np.ndarray
+        A vector corresponding to truth values for class labels of ``dataset``
+    as_regressor: Optional[bool] = None
+        A bool variable used to signify that the ``model``
+        is a regression model. Used to inform the ``scoring_metric``
+    scoring_metric: str
+        The type of scoring measure used to evaluate the predictive perfomance
+        of ``model`` on ``dataset`` - is ``scoring_metric`` is None, the
+        function either applies scikit learn's default
+        scorer for that model if ``model`` is a scikit-learn model,
+        else applies accuracy for a classifier and max error for a regressor.
+    Returns
+    ---------------
+    score: float
+    Float representing the result of applying the ``scoring_metric``
+    (either default or user-inputted) performed on the outputs
+    of  the``model`` predictions on the ``dataset`` and
+    the ``target`` vector."""
+    if SKLEARN_MISSING is True:
+        if as_regressor:
+            predictions = model.predict(dataset)  # type: ignore
+            score = -np.max(np.abs(target - predictions))
```

Comment:
It is not clear to me if "max" should be replace by "mean". Maybe you have a better understanding of why it is more appropriate to use "max" here.